frontend/src/javascript/stage/Painter.js

```diff
@@ -61,4 +61,28 @@ export class Painter {
     getLayer() {
         return this.layers[this.layerPosition];
     }
-}
+
+    rescalePoints(points) {
+        let minX = Math.min.apply(null, points.map(function(point) { return point.x; }));
```

Comment:
Ja bym odizolował Math.min.apply do osobnej funkcji, np. findMinInArray gdzieś do utilsów, albo Array.getMinimumValue. Tak w sumie to dopiero się dowiaduję, że min ma metodę apply, a patrząc na to nie wiadomo do końca co się dzieje (skąd null?)